starport/pkg/cosmosgen/templates/vuex/store/index.ts.tpl

```diff
@@ -74,7 +85,7 @@ export default {
 	},
 	actions: {
 		init({ dispatch, rootGetters }) {
-			console.log('init')
+			console.log('Vuex module: {{ .Module.Pkg.Name }} initialized!')
```

Comment:
I think leaving arbitrary logs in the generated code can be overwhelming since there is no way to disable it.

Reply:
this is a one-off message on initial loading of the app. I think this one is ok.